src/lib/ruby-generator/text2speech.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Define Ruby code generator for Text to Speech Blocks
+ * @param {RubyGenerator} Generator The RubyGenerator
+ * @return {RubyGenerator} same as param.
+ */
+export default function (Generator) {
+    Generator.text2speech_speakAndWait = function (block) {
+        const words = Generator.valueToCode(block, 'WORDS', Generator.ORDER_NONE) || null;
+        return `speak(words: ${words})\n`;
```

Comment:
どのメソッドも、text2speechプレフィックスがないとよくわからないので、追加しましょう。
あと、Rubyタグでコードを入力してみてわかったのですが、 `words:` は長いです。なしでいいです。

```suggestion
        return `text2speech_speak(${words})\n`;
```